### ISSUE TITLE
Extending time on quizzes for students with accomodation

### ISSUE THREAD

**Issue by JulietTeeter** [TWUOnline/bug-tracking#757]
**Which course contains the bug?**  
- Course Code, section, and term (e.g. LDRS 591 OL SP2019)
 Psych 354 02 SP 23
 
**Describe the error and/or your suggestion**  
A clear and concise description of what the error or suggestion is.
 Can someone help me figure out how to extend the ammount of time on a quiz for two students who have academic accomodations?
 
**Location of the bug**  
Please provide a specific link to the page with the error, as well as where the error is on the page:
1. URL:  https://learn.twu.ca/course/view.php?id=27104&section=9#tabs-tree-start
2. Unit #         Topic #         Activity#  
 
**Optional: Screenshots**

**Comment by Jeff-TWU** [comment]
@JulietTeeter I don't believe it is possible to extend time on a Moodle quiz for specific students. We might be able to duplicate the quiz and have it visible for those specific students @barishgolland @cmadland  Do either of you have more wisdom for solving this?

**Comment by kmarjanovic** [comment]
Hi @JulietTeeter and @Jeff-TWU .

We did this last year for PSYC 354 as well...I'm just trying to find the issue & instructions.  @cmadland set it up.

**Comment by kmarjanovic** [comment]
Here are the instructions: https://create.twu.ca/help/moodle/faculty/quizzes/extra-quiz-time
Juliet, please let us know if you'd like us to help with this.

**Comment by JulietTeeter** [comment]
Thank you for this!


Juliet Teeter, MA

Part time instructor of Psychology

Trinity Western University




________________________________
